test/provenanceUtils.test.js

```diff
@@ -0,0 +1,47 @@
+const { resolveSchema } = require('@projecttacoma/node-fhir-server-core/dist/server/utils/schema.utils');
```

Comment:
```suggestion
const { resolveSchema } = require('@projecttacoma/node-fhir-server-core');
```

I believe this is exported by the root of `node-fhir-server-core`